ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1094 @@
+The second project represents the called side. This project is intended to be deployed
+to the other two servers (`server2` and `server3`). The two projects should not be deployed to the same server.
+
+.Maven projects in this quickstart
+[cols="40%,60%",options="headers"]
+|===
+|Project |Description
+
+|`client`
+|An application that you deploy to the first server, to `server1`.
+It provides REST endpoints that provides EJB remote calls to the `server application`
+residing on the other servers.
+Besides the remote EJB call the REST endpoint invocation processes data insertion to a database.
+As the REST invocation starts a transaction then the transaction enlists two participants
+(the database and the remote server called with EJB remote invocation).
+The transaction manager then uses the two-phase commit for committing the transaction.
+The quickstart examines failures and shows the transactional behaviour
+in such cases.
+
+|`server`
+|An application that receives the remote EJB calls from the `client` application deployed at `server1`.
+This application is deployed to `server2` and `server3`.
+The EJB component receives the EJB remote call and depending on the scenario
+it resumes on the transaction propagated in the context of the remote EJB call.
+The call inserts data to the database.
+The applications also enlists a special XAResource to the transaction to demonstrate the quickstart.
+|===
+
+== Overview of running the quickstart
+
+The description is separated into two parts. The first one explains how to run the quickstart in a bare metal environment.
+The second one talks about deployment of the quickstart to Kubernetes and to OpenShift.
+
+=== Running the quickstart in a bare metal environment
+
+. <<_setup_the_environment_, Copy the {productName} installation directory into three directories>> `server1`, `server2` and `server3`.
+. <<_configure_the_credentials_for_remote_call_authentication, Configure credentials>> on `server2` and `server3`.
+. <<remote_configuration_cli, Configure a remote outbound connection>> at `server1` to both servers `server2` and `server3`.
+. <<_configure_datasources, Configure datasources>> first on `server1` and then on both servers `server2` and `server3`.
+. <<_start_productname_servers, Start all 3 servers>>.
+. <<_running_the_quickstart, Compile and deploy the quickstart applications>> to all servers.
+.. The deployed application at `server1` behaves as a client to both other servers.
+   The client server, server1, offers <<rest-endpoints, several HTTP endpoints>>.
+   When you invoke the HTTP endpoint the EJB remote call is executed to `server2` or `server3`.
+. <<_examining_the_quickstart, Call one of the HTTP endpoints>> at `server1`.
++
+Based on the scenario EJB remote calls hit the remote server and
+specific behaviour can be observed (see<<rest-endpoints, list of the quickstart scenarios>>).
```

Comment:
```suggestion
you can observer specific behavior (see<<rest-endpoints, list of the quickstart scenarios>>).
```
Avoid passive voice.
Red Hat technical content follows the IBM style guide, which recommends American spelling and usage.